### ISSUE TITLE
WeaponModModule: Fix for global values not working as intended

### ISSUE THREAD

**Issue by the-almighty-carl** [simon-wh/PAYDAY-2-BeardLib#211]
Supposedly the latest stable release of BeardLib included the tweaks to the WeaponModModule script that would make global values work correctly for weapon mods with "drop" set to true (read: #208). However, all this fix has done is change the behavior of the global value bug instead of fixing it. Now instead of the weird behavior depending on what global value/dlc parameters are set and what ones aren't, it behaves consistently in its malfunctioning - if you have any parameter related to dlcs or global values set on a weapon mod it creates two different instances of said weapon mod, one of which has the intended global value but makes the weapon unequipable due to DRM locking, and the other of which doesn't have the intended global value but doesn't make the weapon unequipable.

Worse still, it seems that the WeaponModModule global value fix actively broke a lot of the WeaponModModule's core functions. I'm seeing reports of people not being able to equip custom weapon mods across the board (#209, #210), and while I'd have to dig deeper to see if it truly affects every single custom weapon mod out there I did notice that a few seemingly random weapon mods that were once freely equipable are no longer available to me.

tl;dr Probably another one of those times where you fix one bug and it creates thirty new bugs. Ouch.

**Comment by the-almighty-carl** [comment]
Update: Someone dicking around with one of my mods blundered into a bit of an interesting discovery. Turns out that, the weapon mods that suddenly become unavailable are weapon mods with drop values set to false, and setting their drop values to true magically makes them available again.

Not good enough at Lua, especially not in the context of this game, to make any guess about what this implies, but I have a feeling that if someone can figure out why that fixes the issue and how it relates to the issue itself it'd be a great place to start working on a fix.
